### PR TITLE
Port `dcos marathon app kill` command to Go

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Review by janisz — CHANGES_REQUESTED** [review]
Please add tests

File: pkg/cmd/marathon/app/app_kill.go [hunk 1]
```diff
@@ -13,12 +16,45 @@ func newCmdMarathonAppKill(ctx api.Context) *cobra.Command {
...
+
+				enc := json.NewEncoder(ctx.Out())
+				enc.SetIndent("", "    ")
+				return enc.Encode(deployment)
+			}
+
+			tasks, err := client.KillTasks(args[0], scale, host)
+			b, _ := json.Marshal(tasks)
+			fmt.Printf("Killed tasks: [%s]\n", string(b))
+			return nil
```
Comment (br-lewis) [inline pkg/cmd/marathon/app/app_kill.go]: The linter caught that you should be returning `err` here.

**Review by janisz — APPROVED** [review]

File: pkg/cmd/marathon/app/app_kill.go [hunk 2]
```diff
@@ -13,13 +17,62 @@ func newCmdMarathonAppKill(ctx api.Context) *cobra.Command {
...
+	cmd.Flags().BoolVar(&scale, "scale", false, "Scale the app down after performing the operation.")
 	cmd.Flags().StringVar(&host, "host", "", "The hostname that is running app.")
 
 	return cmd
 }
+
+func marathonAppKill(ctx api.Context, client marathon.Client, appID string, scale bool, host string) error {
+	if scale {
+		_, err := client.API.Application(appID)
+		if apiErr, ok := err.(*goMarathon.APIError); ok && apiErr.ErrCode == goMarathon.ErrCodeNotFound {
```
Comment (makkes) [inline pkg/cmd/marathon/app/app_kill.go:46]: What if there's another error? Shouldn't we exit in those cases as well?

**Review by makkes — CHANGES_REQUESTED** [review]

File: pkg/cmd/marathon/app/app_kill.go (hunk 2 above)
Comment (janisz) [reply pkg/cmd/marathon/app/app_kill.go:46]: :+1: @ArmandGrillet Can you add a test for this case?

**Review by makkes — APPROVED** [review]

**Pull request merged**